README.md

```diff
@@ -38,7 +38,7 @@ terminal (Mac OS), `oh-my-zsh` with `powerlevel9k` theme and `powerline nerd-fon
   ```sh
   function lc()
   {
-    ruby /path/to/colorls/colorls.rb $1;
+    ruby ~/bin/colorls/colorls.rb $1;
```

Comment:
Can you also add a comment to the `lc` function (after line 41) regarding how to use with `-r` flag option as default, with `ruby ~/bin/colorls/colorls.rb $1 -r;`?